src/controllers/wing.controller.ts

```diff
@@ -112,4 +112,53 @@ const getAllWings = async (req: Request, res: Response) => {
   }
 };
 
-export { createWing, getWing, updateWing, deleteWing, getAllWings };
+const addCordinator = async (req: Request, res: Response) => {
+  try {
+    const response = await wingServices.addCordinator(req.params.id, req.body);
+    return res.status(200).json({
+      data: response,
+      message: "Coordinator added successfully",
+      success: true,
+      err: {},
+    });
+  } catch (error: any) {
+    return res.status(501).json({
+      data: {},
+      message: error?.message || "Failed to add Coordinator",
+      success: false,
+      err: error,
+    });
+  }
+};
+
+const updateCordinator = async (req: Request, res: Response) => {
```

Comment:
I don't think so there is need of update coordinators instead what we can do is to make a route for delete co-ordinator.
If we want to update the coordinator each  season we can do easily by removing and adding.

I will prefer to go with remove co-ordinators and add-coordinators only.

Reply:
Yeah that makes sense. 
Let's go with Add and Delete only then..